oranda-css/css/pages/artifacts.css

```diff
@@ -12,35 +12,26 @@
 
 .artifacts {
   @apply sm:flex items-center flex-col mb-8 p-0 hidden;
-  color: var(--light-highlight-fg-color);
-  background-color: var(--light-highlight-bg-color);
+  color: var(--highlight-fg-color);
+  background-color: var(--highlight-bg-color);
 }
 
 .artifacts-table {
   @apply block max-w-full overflow-auto;
 }
 
-.dark .artifacts {
-  color: var(--dark-highlight-fg-color);
-  background-color: var(--dark-highlight-bg-color);
-}
-
 ul.tabs {
   @apply flex border-b-2;
+  border-color: var(--highlight-fg-color);
```

Comment:
don't see an equivalent change this is mirroring, what's up..?

Reply:
this fixes an implied default border color that was looking pretty bad in axolight